src/contracts/Permission/PermissionCheckerInterface.php

```diff
@@ -19,20 +19,26 @@ public function getRestrictions(array $hasAccess, string $class): array;
     public function canCreateInLocation(Location $location, $hasAccess): bool;
 
     /**
-     * @internal
-     *
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\BadStateException
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     *
```

Comment:
Usually, we don't have an empty line before `@internal`

```suggestion
```

Reply:
If we remove this empty line CS fixer will fail